src/domains/event/repository/event.repository.impl.ts

```diff
@@ -35,6 +36,7 @@ export class EventRepositoryImpl implements EventRepository {
       take: 1,
       where: {
         issueId,
+        deletedAt: null,
```

Comment:
If possible, add a way to verify if the issue has not been logically deleted in createIssueChangeLog,  createIssueBlockEvent, createTimeTrackingEvent, createManualTimeModification methods. If not possible, add a verification in service to ensure we don`t create or modify data related to the issue.

Reply:
Done.